server/src/test/java/org/solarbank/server/integration/EnergySavingControllerIT.java

```diff
@@ -20,6 +23,9 @@ public class EnergySavingControllerIT extends IntegrationTestBase {
     @SpyBean
     private CalculateService calculateService;
 
+    @SpyBean
+    private NasaClient nasaClient;
+
     @Test
     public void energySavingController_validRequest_status200() {
```

Comment:
Uses the set user input thats in the IT test base and expects the correct response